### ISSUE TITLE
Update website colors to comply with color contrast guidelines.

### ISSUE THREAD

**Issue by getorganizedbk** [GetOrganizedBrooklyn/getorganizedbk-website#4]
Colors throughout the website are inaccessible and do not comply with WCAG color contrast guidelines. 

For reference: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

Update colors in the default styles, Home Page, and Working Group pages to match the following visual: https://www.figma.com/file/WHVnX8xXlIt3gd2r0rkR1zbk/GetOrganizedBK-Website?node-id=12%3A3

NOTE: 
1) Ideally we should set colors in the WYSIWYG Default Styles as much as possible, instead of on individual pages. (Appearance > Customize)
2) We may then need to go to individual pages and undo any color overrides in the modules that are conflicting with the above default style updates.  (In the Divi Builder on each page > hamburger menu on each module > Design tab > click the rewind icon next to the specific color setting)
3) Manual CSS (Appearance > Edit CSS) should be a last resort.

### Primary Buttons: 
Text color: FFF (white)
Button color: 1A6A75 (teal)
Hover color: 433B4F (dark blue)

### Home Page secondary buttons (working group names):
Text color: #433B4F (dark blue)
Hover color: #21212C (black)

### Tertiary buttons and links: 
Text color: #1A6A75 (teal)
Hover color: #433B4F (dark blue)

### Light blue backgrounds on home page
Background color: #E5F4EF (lightest blue)

### Footer background
Background color: #931033 (dark red)

## Text
All black text: #21212C (black)
All white text: #FFF (white)

![gobk style guide summer 2018](https://user-images.githubusercontent.com/28609124/44672887-dd52af80-a9f7-11e8-976b-908ae5926af4.jpg)

**Comment by anniehirshman** [comment]
- updated default style colors throughout Appearance > Customize panel (except buttons)
- updated hex values in existing CSS (Appearance > Additional CSS)

Still to do:
- update buttons throughout
- update home page backgrounds 
- update footer colors
- check pages for any remaining individual overrides

**Comment by anniehirshman** [comment]
Updated home page backgrounds. 

Footer colors are now accessible, but should be updated to the latest design.  See Issue #10 . 

Button colors still need to be updated.  See Issue #9
